tests/tests/operation_tests.cpp

```diff
@@ -3220,16 +2849,6 @@ BOOST_AUTO_TEST_CASE(escrow_release_validate)
         op.agent = "sam";
         op.receiver = "bob";
 
-        BOOST_TEST_MESSAGE("--- failure when scorum < 0");
```

Comment:
why you delete this?

Reply:
ok, this test will be restored. But I will fix asset class to receive only SCR or SP symbols.